front/migrations/20240620_backfill_user_message_origin.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE user_messages SET "userContextOrigin" = 'slack' WHERE "userContextOrigin" IS NULL AND "userContextProfilePictureUrl" ILIKE '%slack%';
```

Comment:
Why not writing a script? We have a lot of rows in this table, not a big fan of performing big SQL write queries like this!

Reply:
We have 1.5m rows. My take is that SQL will probably be the most efficient at scanning and updating vs a gazilion back and forth with a script.

The question is is it risky. Given the nature of the query (fine to fail, fine to retry) I think it's not

Reply:
Ok your decision.

Reply:
Query took 2s :)

Reply:
Will have to re-run it post deploy